holo-yang/modules/augmentations/holo-ospf.yang

```diff
@@ -1,309 +1,367 @@
 module holo-ospf {
-  yang-version 1.1;
-  namespace "http://holo-routing.org/yang/holo-ospf";
-  prefix holo-ospf;
+    yang-version 1.1;
+    namespace "http://holo-routing.org/yang/holo-ospf";
+    prefix holo-ospf;
```

Comment:
Somehow my YANG linter (https://github.com/TypeFox/yang-lsp) re-wrote half the file. I will revert that.

Reply:
Certainly. Let's stick with two-space indentation consistent with the IETF models.